packages/extension/src/newtab/App.tsx

```diff
@@ -106,6 +117,7 @@ function InternalApp(): ReactElement {
   const isPageReady =
     (growthbook?.ready && router?.isReady && isAuthReady) || isTesting;
   const shouldRedirectOnboarding = !user && isPageReady && !isTesting;
+  const isFirefox = checkIsBrowser(UserAgent.Firefox);
```

Comment:
Why don't you use the existing one?
`isFirefoxExtension`?

I think in this new flow we don't auto spool webapp right?

Reply:
Otherwise remember nensi also added some more extended browser detection one time.

Reply:
Right. Makes sense, we can actually use that to simplify.